### ISSUE TITLE
Lenovo C14 ChromeOS stopped getting OS update while other ChromeOS devices haven't

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#1112]
In early 2024 Google announced any ChromeOS device that is released after 2022 will get 10 years of software update. Having seen a [review website](https://www.aboutchromebooks.com/news/lenovo-thinkpad-c14-chromebook-review-potent-power-at-a-price/) for C14 published in 2022/12, I assume C14 is newer so it's supposed to receive OS updates. But clicking OS update button doesn't seem to trigger an upgrade and just shows an older OS version number in 129 range (other devices show 130.0.x).

**Comment by 130s** [comment]
Says:
> This device will get automatic software and security updates until June 2032. [Learn more](https://support.google.com/chrome/a?p=auto-update-policy)

**Comment by 130s** [comment]
Ah, it subscribes to stable channel. Will check the channels my other devices subscribe to.

**Comment by 130s** [comment]
Hm, c13-morph also sub's to stable and it has just gotten 130.0.6723.101.
Maybe ChromeOS' release process for c14 is slower?

**Comment by 130s** [comment]
Just as an experiment, I switched the (software upgrade) channel to `Beta` then ChromeOS finally started upgrading. After reboot I found the version to be 131(.0.6779.29), which is way more advanced than my other devices that subscribe to `stable` channel. I'm at least reassured that brya can get an upgrade.

Experiment done so want to roll back then switching back to `stable` would warn me "`Powerwash`" that would erase users is needed, which I don't want to bother doing just now (I'm totally happy with the current state of config on this computer, and there are 2 other user accounts set up already).

So I guess brya is stuck with `beta` channel? Are therey no ways to roll back without Powerwashing?
